### DISCUSSION TITLE
Disable onboard speaker for y20ga cameras

### DISCUSSION THREAD

**Discussion by latigreplata** [roleoroleo/yi-hack-Allwinner#212] [Ideas]
Just a thought I had, based on a personal need;
Could it be possible to disable the speakers in the camera with a software switch in the configuration UI?
I'm using one of my cameras as a nanny cam for our youngest. And if I need to restart the camera for some reason, the camera, with its "Welcom to YI Hong Camera. Waiting to connect. Connected to Wifi!", will 10/10 wake the baby.

Of course I could probably remove the speaker by force, but If I could just disable the speaker in the configuration I would be very happy.

Otherwise, the cameras are working sublimely! Great work on the custom job :)

**Comment by roleoroleo** [comment]
Why does your cam speak? My cam doesn't.
Normally the cam speaks only when wifi is not configured.

**Reply by latigreplata** [reply]
Ah yes. I still havent tried the wifi sd-card workaround. I assumed it spoke every time it rebooted. Will try the workaround this weekend.
Thanks for the reply 👍
